test/clj/game/cards/assets_test.clj

```diff
@@ -2496,6 +2496,7 @@
       (take-credits state :corp)
       (take-credits state :runner)
       (take-credits state :corp)
+      (end-phase-12 state :runner)
```

Comment:
why is this needed?

Reply:
Two sources of draw are triggering at the same time (both crowdfundings), and they're both flagged with runner-phase-12-draw.

Reply:
Should you choose one of the triggers instead?

Reply:
true, didn't consider that.

From my testing, it seems that they do both automatically fire when phase-1.2 is ended, but I didn't know that and it wasn't apparent when looking at the test - so I added in lines to that effect.

Reply:
oh yeah, that's right, it pauses if you want to manually choose. lmao our start of turn stuff is dumb.